src/index.js

```diff
@@ -52,7 +55,29 @@ export default function auth(config = {}) {
     }
 
     // Merge and flatten options
-    const authOptions = Object.assign({}, app.get('auth'), defaults, config);
+    const authOptions = Object.assign({}, defaults, app.get('auth'), config);
+
+    // If we should redirect on success and the redirect route is the same as the
+    // default then we'll set up a route. Otherwise we'll leave it to the developer
+    // to set up their own custom route handler.
+    if (authOptions.shouldRedirectOnSuccess && authOptions.successRedirect === defaults.successRedirect) {
```

Comment:
Does it need `shouldRedirectOnSuccess` or could you just use `successRedirect = false`?